biom/table.py

```diff
@@ -522,6 +522,39 @@ def matrix_data(self):
         """The sparse matrix object"""
         return self._data
 
+    def to_pandas(self):
+        """
+        Returns
+        ----------
+        counts : pd.SparseDataFrame
+            Pandas representation of counts
+        sample_metadata : pd.DataFrame
+            Pandas representation of the sample metadata
+        observation_metadata : pd.DataFrame
+            Pandas representation of the observation metadata
+
+        Note
+        ----
+        This can only handle
+        """
```

Comment:
pandas currently is not a dependency. we can either make it a dependency or make it optional by just checking in this method for whether the import is possible